assets/js/analyzer.js

```diff
@@ -143,7 +143,7 @@ function populateSecondaryAnalyzerList() {
 }
 
 function analyze() {
-    if($('#primaryAnalyzerMode').val() === null) {
+    if($('#primaryAnalyzerMode').val() === null || $('#morphAnalyzerInput').val() === '') {
```

Comment:
Um, looking at the code, both analyzer and generate modes have `required` as a part of HTML. Should I implement a similar check as I have done for `analyzer` mode in the `generate` mode?